profiles/common/modules/features/contact_form/contact_form.install

```diff
@@ -35,4 +35,42 @@ function contact_form_disable() {
   menu_link_delete(NULL, 'contact');
   // Activation message.
   drupal_set_message(t('Contact form feature is now disabled on your site.'));
+  module_disable(array('contact'), FALSE);
```

Comment:
Still not convinced about this sorry :( the issue says:

> Can the default alias be unset, or can the contact module be made optional.

So we should just do that, meaning removing it from `cce_basic_config` is all we need to do, then they can disable it manually on the info site since, I guess, `contact_form` module it's not enabled on their site.

Running `module_disable(array('contact'), FALSE);` is safe but it's still a strange behaviour that might rise questions down the road, I would just remove it, unless I'm missing something.